cmd/oracled/cmd/register_oracle.go

```diff
@@ -64,22 +66,50 @@ func registerOracleCmd() *cobra.Command {
 			defer queryClient.Close()
 
 			// get oracle account from mnemonic.
-			_, err = panacea.NewOracleAccount(conf.OracleMnemonic, conf.OracleAccNum, conf.OracleAccIndex)
+			oracleAccount, err := panacea.NewOracleAccount(conf.OracleMnemonic, conf.OracleAccNum, conf.OracleAccIndex)
 			if err != nil {
 				return fmt.Errorf("failed to get oracle account from mnemonic: %w", err)
 			}
 
 			// generate node key and its remote report
-			_, nodePubKeyRemoteReport, err := generateSealedNodeKey(nodePrivKeyPath)
+			nodePubKey, nodePubKeyRemoteReport, err := generateSealedNodeKey(nodePrivKeyPath)
 			if err != nil {
 				return fmt.Errorf("failed to generate node key pair: %w", err)
 			}
 
 			report, _ := enclave.VerifyRemoteReport(nodePubKeyRemoteReport)
-			_ = hex.EncodeToString(report.UniqueID)
+			uniqueID := hex.EncodeToString(report.UniqueID)
 
 			// request register oracle Tx to Panacea
-			// TODO: add register-oracle Tx
+			oracleCommissionRate, err := sdk.NewDecFromStr(flagOracleCommissionRate)
```

Comment:
I think you can get flag data by `cmd.Flags.GetString(flagOracleCommissionRate)`.

Reply:
And, there is flags in `client/flags/flags.go`. You can add flags to here.

Reply:
Fixed in [fbd3f92](https://github.com/medibloc/panacea-oracle/pull/21/commits/fbd3f92755daaeeb436f0910f2c67a81b30fb503).